### PR TITLE
neotoppm: add page

### PR DESCRIPTION
**Pull request by debghs**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/neotoppm.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# neotoppm
+
+> Convert an Atari Neochrome .neo file into a PPM image.
```
Comment (gutjuri) [inline pages/common/neotoppm.md:3]:
```suggestion
> Convert an Atari Neochrome NEO file into a PPM image.
```

File: pages/common/neotoppm.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+> More information: <https://netpbm.sourceforge.net/doc/neotoppm.html>.
+
+- Generate the PPM image as output for an Atari Neochrome .neo file as input:
```
Comment (gutjuri) [inline pages/common/neotoppm.md:6]:
```suggestion
- Generate the PPM image as output for an Atari Neochrome NEO file as input:
```

**Review by gutjuri — COMMENTED** [review]
Thanks! I added some suggestions.

**Review by gutjuri — APPROVED** [review]

**Comment by debghs** [comment]
@kbdharun can you please review this pr??

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**